app/assets/stylesheets/reviews.scss

```diff
@@ -6,6 +8,151 @@
   border-radius: 0;
 }
 
+/* My Reviews CSS */
+
 #review-label {
   padding-bottom: 10px;
-}
\ No newline at end of file
+}
+
+.myreview-container {
+  background: #FFFFFF;
+  box-shadow: 0px 1px 2px 0px #B8B8B8;
+  border-radius: 4px;
+  margin-left:   0px;
+  margin-right:  0px;
+  margin-top:    5px;
+  margin-bottom: 5px;
+  padding: 40px 40px;
+}
+
+#addreview-container {
```

Comment:
exactly -- ids are fair game in js. I think the guidelines I linked advocate that? Or at least having separate identifiers/classes for javascript related behavior.